homeassistant/components/system_log/strings.json

```diff
@@ -0,0 +1,37 @@
+{
+  "services": {
+    "clear": {
+      "name": "Clear all",
+      "description": "Clears all log entries."
+    },
+    "write": {
+      "name": "Write",
+      "description": "Write log entry.",
+      "fields": {
+        "message": {
+          "name": "Message",
+          "description": "Message to log."
+        },
+        "level": {
+          "name": "Level",
+          "description": "Log level."
+        },
+        "logger": {
+          "name": "Logger",
+          "description": "Logger name under which to log the message. Defaults to `system_log.external`."
+        }
+      }
+    }
+  },
+  "selector": {
+    "level": {
+      "options": {
+        "debug": "Debug",
```

Comment:
These are the same ones as for `logger`. Perhaps not good to do cross-integration links but on this occasion it might be ok?

Reply:
They are indeed cross-integration, which we should avoid.